compiler/optimizer/VPHandlers.cpp

```diff
@@ -8142,6 +8152,11 @@ static TR::Node *constrainNarrowIntValue(OMR::ValuePropagation *vp, TR::Node *no
    const int64_t min = -signBit;
    const int64_t max = signBit - 1;
 
+if (vp->trace())
+{
+traceMsg(vp->comp(), "In constrainNarrowIntValue:  resultSizeBits == %d; signBit == %d; padding64 == %d; min == %d; max -- %d\n", resultSizeBits, signBit, padding64, min, max);
```

Comment:
This (and the below) tracing seems to have been unintentionally left here (e.g. it isn't indented)

Reply:
Sigh!  I'm sorry about that.  I've fixed it and force pushed the change again under commit 43ba52729a70112519803f92467cd844cc313c5b.